### ISSUE TITLE
The training results are not very good

### ISSUE THREAD

**Issue by Kyouii0531** [Robust-Jay/PyTorch-QAE-for-Low-Dose-CT-Denoising#1]
I tried using this QAE model for training, but I found that the loss cannot be reduced during the training process. After multiple epochs of training, the PSNR has maintained a level similar to that of LDCT, so the effect is not very good. If you can train smoothly and achieve the correct results, can you give me some tips? thank you!
